controllers/registry_test.go

```diff
@@ -209,50 +209,6 @@ func TestDPAReconciler_getSecretNameAndKeyforBackupLocation(t *testing.T) {
 
 			wantProfile: "aws-provider-no-cred",
 		},
-		{
```

Comment:
why these tests were removed?

Reply:
Only keeping the velero tests in this function, will write a separate Test function for CloudStorage tests, as both the functions are separate now

Reply:
saw the comment that it will be included later, but I believe this should not be a separate PR

Reply:
Added.